frontend-react/src/pages/daily/Table/ReportsTable.tsx

```diff
@@ -1,64 +1,83 @@
-import { useResource } from "rest-hooks";
-import { SetStateAction, useEffect, useMemo, useState } from "react";
-import { useOktaAuth } from "@okta/okta-react";
+import { Dispatch, SetStateAction, useEffect, useMemo, useState } from "react";
 
 import { getUniqueReceiverSvc } from "../../../utils/ReportUtils";
-import ReportResource from "../../../resources/ReportResource";
 import Table, { TableConfig } from "../../../components/Table/Table";
-import { getStoredOrg } from "../../../contexts/SessionStorageTools";
-import useFilterManager from "../../../hooks/filters/UseFilterManager";
-import { PageSettingsActionType } from "../../../hooks/filters/UsePages";
+import useFilterManager, {
+    FilterManagerDefaults,
+} from "../../../hooks/filters/UseFilterManager";
+import Spinner from "../../../components/Spinner";
+import { useReportsList } from "../../../hooks/network/History/ReportsHooks";
+import { TempRSReport } from "../../../network/api/History/Reports";
+import { useSessionContext } from "../../../contexts/SessionContext";
+import { showError } from "../../../components/AlertNotifications";
 
 import TableButtonGroup from "./TableButtonGroup";
 import { getReportAndDownload } from "./ReportsUtils";
 
+const filterManagerDefaults: FilterManagerDefaults = {
+    sortDefaults: {
+        column: "sent",
+        locally: true,
+    },
+};
+
+interface ReceiverFeeds {
+    activeFeed: TempRSReport[] | undefined;
+    setActiveFeed: Dispatch<SetStateAction<string | undefined>>;
+    feeds: string[] | undefined;
+}
+const useReceiverFeeds = (reports: TempRSReport[]): ReceiverFeeds => {
```

Comment:
This is scoped in the same module because it isn't reused elsewhere, but the logic was complex enough that it should not live inside our rendering component

Reply:
Still covered amply by test coverage according to SonarCloud; win! :D

Reply:
Yeah, exporting JUST for tests is tricky. I did something, somewhere else in this project to export for tests, but I can't find it again of course.

Reply:
Found it. Search for `_exportForTesting`

Reply:
I think my philosophy is it'll be exported if it needs to be tested. Here, there's no need since it's just a single-use abstraction of logic and the integration test between the component and that state works as coverage :)